tests/transforms/test_torchvision_transforms.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import torch
+from PIL import Image
+
+from lightly.transforms import ToTensor
+
+
+def test_ToTensor() -> None:
+    img_np = np.random.randint(0, 255, (20, 30, 3), dtype=np.uint8)
+    img_pil = Image.fromarray(img_np)
+    img_tens = ToTensor()(img_pil)
+    assert isinstance(img_tens, torch.Tensor), "Expected torch tensor"
+    assert img_tens.shape == (3, 20, 30), "Wrong shape"
+    assert img_tens.dtype == torch.float32, "Wrong dtype"
+    assert img_tens.max() <= 1.0 and img_tens.min() >= 0.0, "Wrong range"
```

Comment:
Two small things:
1. We usually import classes `from torch import Tensor` and then refer to them only via the class name without the module name. For functions we import the module name and the call `module.function()`
2. Don't overwrite the assertion messages (e.g with `"wrong shape"`). Pytest automatically generates detailed assertion messages that are lost if you overwrite them. For simple tests like this I would not mention the "wrong shape", "wrong dtype" etc.